docs/src/pages/components/data-grid/group-pivot/group-pivot.md

```diff
@@ -112,13 +112,95 @@ You can limit the sorting to the top level rows with the `disableChildrenSorting
 
 {{"demo": "pages/components/data-grid/group-pivot/TreeDataFullExample.js", "bg": "inline", "defaultCodeOpen": false}}
 
-## 🚧 Master detail [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
+## Master detail [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
```

Comment:
You can also remove the 🚧 from the [feature comparison](https://deploy-preview-3387--material-ui-x.netlify.app/components/data-grid/getting-started/#feature-comparison)

For [Pricing page](https://mui.com/pricing/) I don't know if it is on X or core repo

Reply:
To update the Pricing page it needs to be another PR in the core repo.